src/util/base64.h

```diff
@@ -1,26 +1,37 @@
 /*
-Minetest
```

Comment:
this one is a minetest source

Reply:
No, Minetest is not the author or owner of the code